Tasks/Common/VstsAzureHelpers_/make.json

```diff
@@ -0,0 +1,11 @@
+{
+    "externals": {
+        "archivePackages": [
+            {
+                "archiveName": "openssl.zip",
```

Comment:
Does this bring any change to TPN?

Reply:
Yes, right. We should update the ThirdPartyNotices.txt of all tasks.

Reply:
Can you include those chanages.